doc/design/cluster_train/data_dispatch.md

```diff
@@ -92,11 +92,11 @@ random_images-00099-of-00099
 
 #### 进行训练
 
-PaddlePaddle提供专用的[data reader creator](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/reader/README.md#python-data-reader-design-doc)，生成给定SSTable文件对应的data reader。**无论在本地还是在云端，reader的使用方式都是一致的**：
+PaddlePaddle提供专用的[data reader creator](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/reader/README.md#python-data-reader-design-doc)，生成给定RecordIO文件对应的data reader。**无论在本地还是在云端，reader的使用方式都是一致的**：
 
 ```python
 # ...
-reader = paddle.reader.creator.SSTable("/home/random_images-*-of-*")
+reader = paddle.reader.creator.RecordIO("/home/random_images-*-of-*")
```

Comment:
I second both of you @helinwang and @Yancey1989 that from the perspective of user program running in a Pod, it is only I/O with the local filesystem, as 

1. the home directory should have been mapped to the Pod-local directory `/home`,  and 
1. some shared directories, e.g., the pre-downloaded `paddle.v2.dataset` data, should have been mapped to the Pod-local directory `/common`.

and from the perspective of our client tool `paddle`, it has to refer to files in the distributed filesystem in a special format.  But 

- I don't prefer `pfs:///home/$USER/cifa/...`, because if we are going to be compatible with the URL standard,  we need 3 `/`s in above URL.  
- Instead, I prefer `/pfs/$DATACENTER/home/$USER/cifa/...`, which has been used in Google.